go/vt/vtgate/engine/ordered_aggregate.go

```diff
@@ -290,8 +308,8 @@ func (oa *OrderedAggregate) GetFields(ctx context.Context, vcursor VCursor, bind
 	if err != nil {
 		return nil, err
 	}
-
-	_, fields, err := newAggregation(qr.Fields, oa.Aggregates)
+	env := evalengine.NewExpressionEnv(ctx, bindVars, vcursor)
+	_, fields, err := newAggregation(qr.Fields, oa.Aggregates, env)
```

Comment:
If we also create this here and also in other spots, should we create it once inside the `OrderedAggregate`?

Reply:
Ah ok, that makes sense. I guess we don't really have an execution context here we can keep it in to avoid having to allocate it each time?

Reply:
sad but true